src/styles/components/_Header.scss

```diff
@@ -16,7 +16,7 @@
 
 // Add `a` to give a higher specificity. This fixes the grid utility classes taking precedence
 // over this class, which results in `flex-shrink` not getting applied on small width screen.
-a.hc-c-logo-link {
+.hc-c-logo-link.ds-l-col.ds-l-col--auto {
```

Comment:
Can you talk more about this change? The issue that it addresses?

Reply:
- The lint considers the format a.className a violation.  Refer to this article for more info:  https://nicedoc.io/stylelint/stylelint/blob/master/lib/rules/selector-no-qualifying-type/README.md. 

- I'm going to try this format a > .className and see if it does what's expected. The color for the Log in should be blue.

Reply:
the a> .className format did not work and left the css selector as originally submitted